src/main/resources/application.properties

```diff
@@ -13,7 +13,7 @@
 #
 
 spring.cloud.refresh.refreshable=none
-server.port=7750
+server.port=8080
```

Comment:
This change is needed for running Pulsar Manager only using the Java Backend Service.
But we can leave 7750 by now and have a configuration guide

Reply:
Shall we consider changing a port? The port 8080 is easy to cause port conflict?

Reply:
To me another port will be okay.
Please check that port 8080 is required below at zuul.routes.admin.url

I suggest using 8888.
I don't know the amount of changes needed to the docs and docker files

if we move forward with bundling the front-end with the back-end than docker files will be far easier (no need for nginx)